src/index.js

```diff
@@ -9,13 +9,15 @@ function thermostatFactory({ Service, Characteristic }) {
       this.log = log
       this.name = config.name
       this.address = config.address.toLowerCase()
+      this.manualOnly = config.manual || false
       this.discoverTimeout = config.discoverTimeout || (60 * 1000) // 1 minute
       this.connectionTimeout = config.connectionTimeout || (10 * 1000) // 10 seconds
       this.disableBoostSwitch = false
       this.temperatureDisplayUnits = Characteristic.TemperatureDisplayUnits.CELSIUS
 
       this.thermostat = new Thermostat({
         address: this.address,
+        manual: this.manualOnly,
```

Comment:
Sometimes you're using `manual` and sometimes `manualOnly`. Please unify the property names. I think `manualOnly`is more clear.